mergify_engine/engine/__init__.py

```diff
@@ -327,7 +327,7 @@ async def run(
 async def create_initial_summary(
     redis: utils.RedisCache, event: github_types.GitHubEventPullRequest
 ) -> None:
-    owner = event["repository"]["owner"]["login"]
+    owner = event["repository"]["owner"]["id"]
```

Comment:
This clearly cannot work… except if `owner` is never used?

Reply:
It's only used in `clients.github.aget_client()` that why it works, we should rename the variable owner_id to avoid confusion.